operator/cmd/flags.go

```diff
@@ -283,9 +283,6 @@ func init() {
 	flags.String(option.BGPConfigPath, "/var/lib/cilium/bgp/config.yaml", "Path to file containing the BGP configuration")
 	option.BindEnv(Vp, option.BGPConfigPath)
 
-	flags.Bool(option.SkipCRDCreation, false, "When true, Kubernetes Custom Resource Definitions will not be created")
-	option.BindEnv(Vp, option.SkipCRDCreation)
```

Comment:
@pippolo84 just want to double check, should I invoke `option.BindEnv(Vp, client.SkipCRDCreation)` here to bind the env var?

Reply:
No, it will be handled by `cell.Config`.